corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -1497,10 +1588,8 @@ def testOtherUserCloses(self):
         assert_user_has_case(self, self.user, case_id, restore_id=self.sync_log._id)
 
         # make sure closed cases don't show up in the next sync log
-        next_synclog = synclog_from_restore_payload(
-            generate_restore_payload(self.project, self.user, restore_id=self.sync_log._id)
-        )
-        self.assertFalse(next_synclog.phone_is_holding_case(case_id))
+        next_synclog = self.get_next_sync_log(restore_id=self.sync_log._id)
+        assert_user_doesnt_have_case(self, self.user, case_id, restore_id=next_synclog.get_id)
```

Comment:
This was changed to use `assert_user_doesnt_have_case`, which performs a sync before doing the check. See similar comment earlier in this PR for the reason. I have logs from before and after running this test that demonstrate the test was not doing what the comments said it was doing, and can provide that if needed.